### DISCUSSION TITLE
What can you do with an order after creating one? (besides aborting)

### DISCUSSION THREAD

**Discussion by angmnivore** [ahivert/tgtg-python#271] [Q&A]
Since the API doesn't yet support paying for an order, how do you actually proceed with an order? I went to the app and the UI does not show anything related to a reserved order. Any ideas on how to pay, either manually or via code?

**Comment by FrederikvSvane** [comment]
+1
Any updates on this?

**Comment by morpheus9393** [comment]
I'm also wondering how we can pay for the order? I think an order without a way to pay is pretty worthless.
